R/api.R

```diff
@@ -37,10 +37,10 @@ on_load(chat_perform_stream <- coro::generator(function(provider, req) {
 
   repeat {
     event <- httr2::resp_stream_sse(resp)
-    if (is.null(event)) {
-      abort("Connection failed")
-    } else if (stream_is_done(provider, event)) {
+    if (stream_is_done(provider, event)) {
```

Comment:
@jcheng5 I need to think through how this applies to `chat_perform_async_stream()`